lblod/frontend-embeddable-notule-editor
# Add basic cypress tests for the editor and citation-plugin

## Overview
This PR adds three additional cypress tests:
- A test which checks if basic typing in the editor works
- A test which checks if the citation-plugin modal displays at least 1 result
- A test which checks if the filtering of the citation-plugin works as expected

### How to test/reproduce
- Start the dummy app
- Start the cypress suite
- Ensure that all new test runs are successful
